File: Elixir/fizz_buzz/test/fizz_buzz_test.exs
defmodule FizzBuzzTest do
  use ExUnit.Case
  doctest FizzBuzz
  
  test "1 is 1" do
    assert FizzBuzz.run(1) == 1
  end
  
  test "2 is 2" do
    assert FizzBuzz.run(2) == 2
  end

  test "3 is Fizz" do
    assert FizzBuzz.run(3) == "Fizz"
  end
  
  test "5 is Buzz" do
    assert FizzBuzz.run(5) == "Buzz"
  end
  
  test "15 is Fizz" do
    assert FizzBuzz.run(15) == "FizzBuzz"
  end
end

File: python/FizzBuzz/tests/fizzbuzz_test.py
# Run from FizzBuzz folder with: python -m unittest  discover -s tests -p "*_test.py"

import unittest
from src import fizzbuzz

class TestFizzBuzz(unittest.TestCase):
    def test_multiple_of_three(self):
        self.assertEqual(fizzbuzz.process(6), 'Fizz')

    def test_multiple_of_five(self):
        self.assertEqual(fizzbuzz.process(20), 'Buzz')

    def test_fizzbuzz(self):
        self.assertEqual(fizzbuzz.process(15), 'FizzBuzz')

    def test_regular_number(self):
        self.assertEqual(fizzbuzz.process(2), 2)

#if __name__ == '__main__':
    #unittest.main()
suite = unittest.TestLoader().loadTestsFromTestCase(TestFizzBuzz)
unittest.TextTestRunner(verbosity=2).run(suite)